openfe/protocols/openmm_septop/base.py

```diff
@@ -0,0 +1,1436 @@
+            sampler.minimize(
+                max_iterations=settings[
+                    'simulation_settings'].minimization_steps
+            )
+            # equilibrate
+            if verbose:
+                self.logger.info("equilibrating systems")
+
+            sampler.equilibrate(int(equil_steps / mc_steps))  # type: ignore
+
+            # production
+            if verbose:
+                self.logger.info("running production phase")
+            sampler.extend(int(prod_steps / mc_steps))  # type: ignore
+
+            if verbose:
+                self.logger.info("production phase complete")
+
+            if verbose:
+                self.logger.info("post-simulation result analysis")
+
+            analyzer = multistate_analysis.MultistateEquilFEAnalysis(
+                reporter,
+                sampling_method=settings[
+                    'simulation_settings'].sampler_method.lower(),
+                result_units=unit.kilocalorie_per_mole
+            )
+            analyzer.plot(filepath=shared_basepath, filename_prefix="")
+            analyzer.close()
+
+            return analyzer.unit_results_dict
+
+        else:
+            # close reporter when you're done, prevent file handle clashes
+            reporter.close()
+
+            # clean up the reporter file
+            fns = [self.shared_basepath / settings[
+                'output_settings'].output_filename,
+                   self.shared_basepath / settings[
+                       'output_settings'].checkpoint_storage_filename]
+            for fn in fns:
+                os.remove(fn)
+
+            return None
+
+    def run(
+            self, serialized_system, serialized_topology, dry=False, verbose=True,
+            scratch_basepath=None, shared_basepath=None,
+                 ) -> dict[str, Any]:
+        """
+        Run the simulation part of the SepTop protocol.
+
+        Parameters
+        ----------
+        serialized_system: pathlib.Path
+          Path to the serialized OpenMM system
+        serialized_topology: pathlib.Path
+          Path to the serialized topology of the system
+        dry: bool
+          Whether or not to run a dry run
+        verbose: bool
+
+        Returns
+        -------
+        dict : dict[str, str]
+            Dictionary with paths to ...
+        """
+        # 0. General preparation tasks
+        self._prepare(verbose, scratch_basepath, shared_basepath)
+
+        # 1. Get components
+        self.logger.info("Running the SepTop simulation.")
+
+        settings = self._handle_settings()
+        alchem_comps, solv_comp, prot_comp, smc_comps = self._get_components()
+
+        system = deserialize(serialized_system)
+        pdb = simtk.openmm.app.pdbfile.PDBFile(str(serialized_topology))
+        positions = pdb.getPositions(asNumpy=True)
+
+        # Check that the restraints are correctly applied by running a short equilibration
+        equ_positions, box_AB = self._pre_equilibrate(
```

Comment:
```suggestion
        equil_positions, box_AB = self._pre_equilibrate(
```
or "equilibrated" (equ is hard to follow name-wise)